org.eclipse.xtext.xbase.lib/build.gradle

```diff
@@ -6,4 +6,6 @@ dependencies {
 	compile "com.google.guava:guava:$versions.guava"
 }
 
-apply from: "${rootDir}/gradle/gwtSourceProject.gradle"
\ No newline at end of file
+apply from: "${rootDir}/gradle/gwtSourceProject.gradle"
+
```

Comment:
sorry herer buiuld.properties

Reply:
This project has no build.properties. What to do?

Reply:
```
[dietrich@dietrich-thinkpad-x230 xtext-lib]$ find . -name "build.properties"
./releng/org.eclipse.xtend.gwt/build.properties
./releng/org.eclipse.xtext.xbase.lib/build.properties
./releng/org.eclipse.xtext.license/build.properties
./releng/org.eclipse.xtext.gwt/build.properties
```  
i did miss these are the features